packages/wdio-config/src/utils.js

```diff
@@ -7,19 +7,21 @@ const REGION_MAPPING = {
     'eu': 'eu-central-1.'
 }
 
-export function getSauceEndpoint (region) {
-    const dc = region ?
-        typeof REGION_MAPPING[region] !== 'undefined' ?
-            REGION_MAPPING[region] : (region + '.')
-        : ''
-    return `${dc}saucelabs.com`
+export function getSauceEndpoint (region, isRDC) {
+    const dcRegion = REGION_MAPPING[region] ? region : 'us'
+
+    if (isRDC){
+        return `${dcRegion}1.appium.testobject.com`
```

Comment:
this makes it difficult to allow to set arbitrary regions which is important when there will be a new datacenter like `us2`

Reply:
Ok, I understand what you mean, but what if the url will be like the `eu` one. I'd like to keep it simple, if a new DC would be added in the future I'd like to add it then. for now we have a stable solution that would not break if there is a typo and default to the us.
If you'd like it to break it (so not default), then I'm fine with it and revert the change

Reply:
I guess for now it is ok, but I think at some point we need to build a region handler for arbitrary cloud services.